framework/src/components/storage/entities/account.js

```diff
@@ -396,6 +397,9 @@ class Account extends BaseEntity {
 	}
 
 	_getResults(filters, options, tx, expectedResultCount = undefined) {
+		this.validateFilters(filters);
```

Comment:
Are there more cases like this in this entity or any other?

Reply:
Good comment! No, these were the only ones where it was missing for.